### DISCUSSION TITLE
redis is reset after mounted for a while

### DISCUSSION THREAD

**Discussion by T-TRz879** [juicedata/juicefs#2201] [General]
版本：JuiceFS-v1.0.0-beta3
问题在pdf中，请查看：）
[JuiceFS挂载OSS问题.pdf](https://github.com/juicedata/juicefs/files/8879946/JuiceFS.OSS.pdf)

**Comment by T-TRz879** [comment]
挂载失联后redis中的key
<img width="268" alt="image" src="https://user-images.githubusercontent.com/50860504/173093778-4a1e5c06-a896-485c-abf1-f1c67c41fb16.png">

**Comment by davies** [comment]
应该是某种原因导致 Redis 里面的内容被清空了，请检查下是否有脚本在清空它，或者有某种强制淘汰的策略？

**Comment by T-TRz879** [comment]
> 应该是某种原因导致 Redis 里面的内容被清空了，请检查下是否有脚本在清空它，或者有某种强制淘汰的策略？

我查看了redis的log未看出哪里有删除操作。我该具体从哪里着手呢?

**Comment by davies** [comment]
挂载的时候是否有 memory-policy 的告警？

**Comment by T-TRz879** [comment]
挂载的时候没有memory-policy的告警 Redis也没有淘汰策略 服务器内存也是充足的
<img width="275" alt="image" src="https://user-images.githubusercontent.com/50860504/173168973-e003a7e2-1036-4e87-868f-fcc5cee44675.png">

**Comment by davies** [comment]
需要开启一下 AOF，然后当数据被清空后检查 AOF

**Reply by T-TRz879** [reply]
谢谢你的指导。我开启aof尝试一下，看看会不会出现这样的问题：）

**Reply by T-TRz879** [reply]
@davies 开启AOF后，目前未出现这样的问题。谢谢：）
